### DISCUSSION TITLE
Send CAN commands

### DISCUSSION THREAD

**Discussion by goodkiller** [arendst/Tasmota#18213] [General]
Hello, I am very interested about MCP2515 CAN BUS module, but it only reads commands and missing command sending functionality.
Can someone add this functionality?

Thanks in advance.

**Comment by sfromis** [comment]
Sure, it would be possible for *someone* with the hardware, interest and time to add functionality to the driver. You?

**Reply by goodkiller** [reply]
I don't have skills to develope C but i am ready to pay to someone. I need to control battery charger via CAN :)
